src/gebt_poc/model.cpp

```diff
@@ -0,0 +1,25 @@
+#include "src/gebt_poc/model.h"
+
+namespace openturbine::gebt_poc {
+
+Kokkos::View<Section*> CreateSections(std::vector<Section> sections) {
+    auto sections_view = Kokkos::View<Section*>("sections", sections.size());
+    auto sections_host = Kokkos::create_mirror(sections_view);
+
+    for (std::size_t index = 0; index < sections.size(); ++index) {
+        sections_host(index) = sections[index];
+    }
+
+    // *** Question for David: How to implement a deep_copy() for a Kokkos::View<Section*>?
```

Comment:
@ddement Please take a look at this the next time you review it. Looks like we need to implement a `deep_copy()` function for the `Section` class to be able to simply write `deep_copy(sections_view, sections_host)`. Do you think it's something we should look into implementing?

Reply:
On further reflection, it might be smart to not have Views of sections.  Perhaps it would make more sense to just store our matrices as `View<double***>` instances with the first dimension being the number of sections and the second/third dimensions being the matrix sizes.  If it makes sense, we could do a "Sections" class (probably a bad name) to hold the mass, stiffness, and location views.  I think doing a "structure of arrays" rather than "array of structures" in this case will work much better with the GPU.  If you ever want to pass a single matrix into a function, you can use a `Kokkos::subview`.  Does this make sense?  Thoughts?

What you have implemented here wouldn't work on GPU (sections_host can't be accessed on GPU).  We'd either want to implement the deep_copy as you said (I'd have to figure out how to do that), or copy parts of the section into stack memory so that it could be captured up to device.

Reply:
@ddement I finally had a chance to think about the data structure to be used for model creation based on user inputs. I see your point regarding the SoA vs AoS and I agree it will be more efficient for working with GPUs. However, let's remember the `Section`s being created here are just a placeholder for user provided inputs (at some random points) and will be used later on to create the sectional properties at the quadrature points via the interpolation scheme - those are the data structures that will be used in the time integration scheme and we should look into making compatible with GPU. 

For the purposes of this PR, I think we could just create a `std::vector<Section>`, much like some of the recent work where we are leveraging the `std::` containers and algorithms. These are the pre-processing steps and we could run them only in CPU. Does that sound reasonable to you?

@michaelasprague Please take a look and free to let me know if I'm missing something in the above.